src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -7,7 +7,13 @@ namespace System.Runtime.InteropServices
 {
     public static partial class RuntimeInformation
     {
+#if PLATFORM_NETCORE50AOT
+        private const string FrameworkName = ".NET Native";
+#elif PLATFORM_NET46
+        private const string FrameworkName = ".NET Full Framework";
```

Comment:
According to this:
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/project-guidelines.md

>Defines based on conventions should be one of $(OSGroup), $(TargetGroup), $(ConfigurationGroup), or $(Platform), matching exactly by case to ensure consistency.
Examples: <DefineConstants>$(DefineConstants),netcore50</DefineContants>

This:
```c#
#if PLATFORM_NETCORE50
```

should be this:

```c#
#if netcore50
```